omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -22,8 +22,6 @@
 
 	<!-- Add here beans related to the web context -->
 
-	<context:component-scan base-package="org.openmrs.module.edtriageapp.web.controller" />
```

Comment:
Oh, right, this is no longer needed in later versions, correct?

Reply:
Not sure, but there is no such package in the module, so it wouldn't have actually worked ever anyway!